Website/src/items/content/EdgeWebsite/home/speakers.yml

```diff
@@ -10,78 +10,37 @@ SharedFields:
 - ID: "f1a1fe9e-a60c-4ddb-a3a0-bb5b29fe732e"
   Hint: __Renderings
   Value: |
-    <r xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <r xmlns:p="p" xmlns:s="s"
+      p:p="1">
       <d
-        id="{FE5D7FDF-89C0-4D99-9AA3-B5FBD009C9F3}"
-        l="{B103F257-71EF-5392-B6CD-EC8062F68C1C}">
-        <p
-          uid="{A1C3B825-8C7A-5D67-85AF-D877DA0A1816}"
-          key="jss-header"
-          md="{2BDBF7A5-2830-586D-BADB-95A33FA429A3}" />
-        <p
-          uid="{4C73E177-7D69-56C2-86DA-2945966D7A9A}"
-          key="jss-main"
-          md="{83F8CCFF-EDAA-516B-BD68-71A16A354F42}" />
-        <p
-          uid="{8C6B98B5-1E4D-5868-9019-BF85141D3ABC}"
-          key="jss-footer"
-          md="{CC2CF2D1-033E-5D27-BA34-254D30F146CD}" />
-        <p
-          uid="{163E0283-B986-57A3-BEB9-0FDA01E0FCD2}"
-          key="jss-header-content"
-          md="{B0B0F437-11AB-50B9-93D7-9D73EA3C3BD0}" />
-        <p
-          uid="{C679C0A2-921C-5D4C-B3CB-946E0484E6F9}"
-          key="jss-section-content"
-          md="{3ACD7D8D-E5D3-5AF6-BA34-BB869657E602}" />
+        id="{FE5D7FDF-89C0-4D99-9AA3-B5FBD009C9F3}">
         <r
-          rname="Header"
-          id="{0FC536D1-7D7C-579C-953B-454A308B4390}"
-          par=""
-          ph="/jss-header"
-          uid="{3C6A1A92-98F5-509A-8B4D-897041CE042C}" />
-        <r
-          rname="MainNavigation"
-          id="{09516A66-F92F-5638-B073-8F9A79765778}"
-          par=""
-          ph="/jss-header/jss-header-content-{3C6A1A92-98F5-509A-8B4D-897041CE042C}-0"
-          uid="{C2AA52E9-20A3-5AE5-880C-E4F88B051708}" />
-        <r
-          rname="HeroImage"
-          ds="{64D1C128-A38E-525E-A993-2431DE26A39E}"
-          id="{2C13DBC0-415C-5DAA-A7D9-56928F1271C4}"
-          par=""
-          ph="/jss-main"
           uid="{DC97D398-D7B5-5407-B2B2-B28516EB743F}"
-          ccb="Clear on publish" />
-        <r
-          ds=""
-          id="{7AB4158F-135B-5E9B-8106-C4A898F2F02D}"
-          par=""
-          ph="/jss-main"
-          uid="{FEA6367D-427C-44E9-B789-5876A6093332}"
-          ccb="Clear on publish" />
-        <r
-          rname="Section"
-          ds="{174A7AFC-D923-58F3-BEF8-AD8CAAD8C5B9}"
-          id="{2F9D05FD-8E58-56CC-B586-41A3EC155CD7}"
-          par=""
-          ph="/jss-main"
-          uid="{5667C874-385E-5AD2-9D21-57F15F768989}" />
+          p:after="r[@uid='{7CDC810E-AE54-5B4B-BA7D-128CF6F96EDA}']"
+          s:rname="HeroImage"
```

Comment:
The hero of the speakers page has not be updated according to the design with the surfing picture and text at the different breakpoints: https://xd.adobe.com/view/5b52c355-df8c-4ddc-841d-f3dc6cc2342b-0e01/screen/c9fb70da-2f6c-4dea-ba41-95dc511e3d2a/

Reply:
different task created